cmd/edv-rest/startcmd/start.go

```diff
@@ -120,9 +120,15 @@ func createStartCmd(srv server) *cobra.Command {
 				return err
 			}
 
-			databaseURL, err := cmdutils.GetUserSetVar(cmd, databaseURLFlagName, databaseURLEnvKey, true)
-			if err != nil {
-				return err
+			var databaseURL string
+			if databaseType == databaseTypeMemOption {
```

Comment:
looks like there's more going on in this PR than just logging

Reply:
This was done so that when the database type is set to in-memory, the database URL that gets printed is "N/A" instead of a blank string. Just makes it more clear what's going on.